### PR TITLE
Issue 417

### LINKED ISSUE #417: simplify delete rows examples
See https://sagebionetworks.jira.com/browse/SYNPY-810

### PR DESCRIPTION
**Pull request by kdaily**
Fixes #417

### CODE REVIEW

File: articles/tables.md [hunk 1]
```diff
@@ -704,10 +700,10 @@ Clicking on any file will download it.
...
+The faceted navigation on `Tables` (also known as **simple search**) can be used to simplify your search without having to use SQL-like queries. Simple search uses radio buttons and sliders to show all available facets in a menu to the left of the `Table` whereas advanced search employs a SQL-like query to filter the `Table`. To use table facets, navigate to a `Table` or a `File View`. For this example, we will be using the [Synapse Table Demo](https://www.synapse.org/#!Synapse:syn3079449/tables/){:target="_blank"} found in the [Wondrous Research Example](https://www.synapse.org/#!Synapse:syn1901847/wiki/56044){:target="_blank"}. Simple and advanced search both allow you to query for features of interest in a`Table` using different methods.
```
Comment (ychae) [inline articles/tables.md]: "Simple search uses radio buttons and sliders". I think we only use radio buttons and fill-in-the-blank ranges, not sliders. Unless sliders have been there all along and I've somehow missed them??

Comment (kdaily) [reply articles/tables.md]: Good catch, thank you! Can you accept this and I'll file a separate issue to change it separately, please? Not related to this one.

Comment (ychae) [reply articles/tables.md]: of course!

**Comment by kdaily** [comment]
@ychae 

the best

**Pull request merged**